README.md

```diff
@@ -1,9 +1,84 @@
+
+Using the dccmonitor Shiny application requires that the user have a Synapse account, and have permission to access necessary Synapse files. These files include the specific Synapse folder of interest for monitoring (see [Customization](https://github.com/Sage-Bionetworks/dccmonitor/tree/update-readme#customization) for details), along with access to all dccvalidator Synapse dependencies (ex: template and annotation files).
+
+## Installation and Use
+
+dccmonitor can be installed and used in two ways: package installation, and cloning the repository.
+
+### Package Installation
+
+dccmonitor can be installed via devtools:
+
+``` R
+devtools::install_github("Sage-Bionetworks/dccmonitor")
+```
+
+To run the Shiny application, there needs to be a config.yml file in the working directory with the required options (see [Customization](https://github.com/Sage-Bionetworks/dccmonitor/tree/update-readme#customization)). After creating the config.yml file, the application can be run with:
+
+``` R
+library(dccmonitor)
+Sys.setenv(R_CONFIG_ACTIVE = "default") # Replace "default" with the configuration name if not using default.
+run_app()
+```
+
+### Cloning Repository
+
+[Clone the repository](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) to a local directory. Customize the config.yml file, as needed (see [Customization](https://github.com/Sage-Bionetworks/dccmonitor/tree/update-readme#customization)).
+
+If your working directory is the application directory, you can change the app.R file to have your configuration name, if not using the default configuration. Then the following will start the app:
+
+``` R
+renv::restore() # Update project package library
+shiny::runApp()
+```
+
+## Customization
+
+Customizing the application is done via the config.yml file and the specific file validation checks.
+
+### config.yml
+
+A configuration file is required for the application behave correctly. Create a configuration for the application called config.yml (see [this repo's config.yml](https://github.com/Sage-Bionetworks/dccmonitor/blob/master/config.yml) for an example) in your working directory if you have installed the package, or alter the config.yml file in the cloned repository. If not using default as your configuration, set the active configuration as described in the instructions for your choice of installation, [package](https://github.com/Sage-Bionetworks/dccmonitor/tree/update-readme#package-installation) or [cloned repository](https://github.com/Sage-Bionetworks/dccmonitor/tree/update-readme#cloning-repository). Alternatively, if you have forked the repository and added a new set of configuration options to the config.yml file, you can open a Pull Request to see about adding the new options.
```

Comment:
grammatical revision
```suggestion
A configuration file is required for the application to behave correctly. Create a configuration for the application called config.yml (see [this repo's config.yml](https://github.com/Sage-Bionetworks/dccmonitor/blob/master/config.yml) for an example) in your working directory if you have installed the package, or alter the config.yml file if you have cloned the repository. If not using default as your configuration, set the active configuration as described in the instructions for your choice of installation, [package](https://github.com/Sage-Bionetworks/dccmonitor/tree/update-readme#package-installation) or [cloned repository](https://github.com/Sage-Bionetworks/dccmonitor/tree/update-readme#cloning-repository). Alternatively, if you have forked the repository and added a new set of configuration options to the config.yml file, you can open a Pull Request to see about adding the new options.
```

This section is a bit difficult to follow - it may make sense to nest configuration under each installation option.

Reply:
This section is a summarized repeat of the installation instructions specific to the config file. I repeated them in a summarized format since that section is about that file (and in case someone skipped over the instructions altogether). I could remove everything in that paragraph after the first sentence and add about about referring to the instructions.

> A configuration file, called config.yml, is required for the application behave correctly. See the installation instructions above for information on where the file should be located and how to set the environment configuration.

Would this be easier to understand?

Reply:
I think it is better as is, linking to each relevant section. Thanks!